frontend/javascripts/test/model/binary/cube.spec.ts

```diff
@@ -269,3 +269,46 @@ describe("DataCube", () => {
     expect(index).toBe(10570);
   });
 });
+
+// This is not executed in the tests, but can be activated when needed
+// to make performance measurements for getOrCreateBucket
```

Comment:
Maybe add results from your testing to the comment :thinking:? This would help to see whether there were any performance changes. Else one has to checkout other code versions first to be able to check for performance changes

Reply:
I think, there's not much value in committing measurements here, as the numbers will be highly system dependent. When one is interested whether the performance got better or worse, one should execute the benchmark with the old code on the same machine in my opinion.